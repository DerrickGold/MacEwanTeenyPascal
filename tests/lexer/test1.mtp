BeGiN
    WrItE (1, 2 & 3)
EnD.