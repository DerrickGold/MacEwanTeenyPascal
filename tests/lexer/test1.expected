1, TOK_KEY_BEGIN, BeGiN
2, TOK_KEY_WRITE, WrItE
2, TOK_LPAREN, (
2, TOK_NUM, 1
2, TOK_COMMA, ,
2, TOK_NUM, 2
2, TOK_ERROR, &
2, TOK_NUM, 3
2, TOK_RPAREN, )
3, TOK_KEY_END, EnD
3, TOK_PERIOD, .
3, TOK_ENDFILE,
